Add option to calculate settling velocities for differently shaped particles

jormercury opened the pull request:
See:
vignettes/3.2-Particle-characteristics.md (https://github.com/rivm-syso/SBooScripts/blob/05c8b25a530d31693d472076cd2f3d882a565ee7/vignettes/3.2-Particle-characteristics.md)
Please when you have time @nadimsaadi add some more description on the approach used to calculate the Settling Velocity in the above vignette.
and
vignettes/Development/20250826_Plastics_Shape_update.md 
https://github.com/rivm-syso/SBooScripts/blob/05c8b25a530d31693d472076cd2f3d882a565ee7/vignettes/Development/20250826_Plastics_Shape_update.md

Commit:
Documentation of changes

Commit:
Now also Runoff for P species, column could be removed in future

Commit:
Add VegInterceptFrac and SizeRunoff

Commit:
Linux vs Windows and ordering

Commit:
Documentation

Commit:
Only distinction between particle and molecular

Commit:
Allow use of initWorld function for SimpleBox installations not in current project

Commit:
Unified initWorld script based on ChemClass

Commit:
Ignore SimpleBox default install location in project

Commit:
Introduce option to use interceptation factor based on particle size

Commit:
Start with updated documentation

Commit:
start documentation

Commit:
trying to run the model without sufficient input data...

Commit:
documentation

jormercury committed:
TStart of testing implementation of shape

jormercury committed:
Documentation update

jormercury committed:
Verification

jormercury committed:
Documentation for adding shape

nadimsaadi committed:
Update 3.2 Particle characteristics.Rmd

nadimsaadi reviewed (commented):
I left a few comments. Let me know what you think.

jormercury commented on vignettes/3.2-Particle-characteristics.md:
```diff
@@ -1,570 +1,481 @@
+  - Defined by `Shortest_side` (minor diameter), `Intermediate_side`
+    (major diameter) and `Longest_side` (height).
+
+These shapes and sizes are used for calculating:
```
in Rstudio, you can 'knit' (make) the md file from the Rmd file. Please provide the edits in the Rmd file. the .md file is overwritten when the Rmd file is knit.

nadimsaadi replied:
Ah great, I tried setting knit directory as the working directory and it worked :) Will commit the Rmd and md files now.

jormercury committed:
Merge branch 'development' into Shape_setvel

jormercury commented on vignettes/3.2 Particle characteristics.Rmd:
```diff
@@ -8,53 +8,120 @@ editor_options:
 
-An example of a process dependent on these characteristics is the Settling Velocity. The Settling Velocity is calculated through the choice of three methods, of which Stokes is the Default.
+An example of a process dependent on these characteristics is the Settling Velocity.
+The Settling Velocity is calculated through the choice of different drag methods (Dioguardi, Swamee, Stokes, Bagheri) of which the original does not include drag.
```
Great, thanks!

jormercury committed:
Knit Particle characteristics vignette

jormercury committed:
some work to be done

jormercury committed:
some hint for running further illustrations of shape update

nadimsaadi committed:
3.2 Particle characteristics

Updated the files and knitted the .md file. Should be good now.

jormercury committed:
Delete .DS_Store

nadimsaadi committed:
Finalized comparison of k values (drag method and shape)

Wrote some code to compare the k values for original and bagheri method, and sphere vs fiber. Knitted the md file

jormercury committed:
Update documentation in dragmethod, shape and testing

jormercury committed:
update comparison drag methods

jormercury committed:
clarify documentation
